service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -47,8 +47,13 @@ private RunLog runToRunLog(Run run) {
   public ResponseEntity<RunLogResponse> getRuns() {
 
     List<Run> queryResults = runDao.getRuns();
-    List<Run> updatedRunResults = smartPoller.updateRuns(queryResults);
-
+    List<Run> updatedRunResults;
+    try {
+      updatedRunResults = smartPoller.updateRuns(queryResults);
+    } catch (Exception e) {
```

Comment:
I like the idea of runs being units which we can try to update individually. We should probably also refactor this to not be a single giant method... (but I was holding this back because it's not really Katrina's fault and might happen anyway as part of a larger refactor)

Reply:
Exceptions are now handled inside `updateRuns`.